xbmc/pvr/PVRDatabase.cpp

```diff
@@ -229,6 +229,30 @@ void CPVRDatabase::UpdateTables(int iVersion)
 
   if (iVersion < 37)
     m_pDS->exec("ALTER TABLE channelgroups ADD iLastOpened integer");
+
+  if (iVersion < 38)
+  {
+    m_pDS->exec("CREATE TABLE channelgroups_new ("
```

Comment:
If you rename the table first and create the new table similar to `sqlCreateTimersTable` you can save replicating the SQL twice.

Reply:
very true, thx